### PR TITLE
feat(components): offerItem and offersList components closes #80

### LINKED ISSUE #80: Create Offers List & Offer Item Components
**Task Description**
Users should be able to view trade offers of various users on the `explore/offers` page. Users should also be able to view the trade offers of a select collector on that collector's profile page.

- [x] Create the `OffersItem` component
- [x] Create the `OffersList` component

### PR DESCRIPTION
**Pull request by Pearlsky**
**Description**
The essence of this was to create the `OfferItem` and `OffersList` components. 

**Overview**
The `OffersList` component consumes data containing several trade offers and maps each to an `OfferItem`, creating a list of `OfferItems`. This applies to both explore & user profile pages

The `OfferItem` takes the data passed, consumes some in its UI, and queries for some more data in some instances to fill up the gaps. like finding the issue number for a comic & its genres.

The `explore/offers` page represents what's on the `OffersList` for the explore page

### CODE REVIEW

File: packages/frontend/src/components/ui/OfferItem/OfferItem.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  tradeOffer,
+}: IOfferItemProps) => {
+  const DUMMY_COMIC_DETAILS = {
+    issue: 8,
+    imageUrl:
+      'https://cdn.marvel.com/u/prod/marvel/i/mg/3/60/6227804f029af/clean.jpg',
+  };
+
+  if (status === 'fulfilled') {
+    return <> </>;
```
Comment (fmtabbara) [inline packages/frontend/src/components/ui/OfferItem/OfferItem.tsx:39]: if you don't want to return anything, return `null`

**Review by fmtabbara — APPROVED** [review]

File: packages/frontend/src/components/ui/OfferItem/OfferItem.tsx (hunk 1 above)
Comment (Pearlsky) [reply packages/frontend/src/components/ui/OfferItem/OfferItem.tsx]: done, thanks @fmtabbara

**Review by TazDeCoder — CHANGES_REQUESTED** [review]
@Pearlsky can you move the `OfferItem` component into the `components/common` folder since you said you intended this component to be used both in the **Explore** and **Collector** pages.

Also, please move the `OfferList` component and move it into the `components/explore` folder and call it `ExploreOffers` and take whatever markup and logic you have in the **Offers** page and delegate that into that component.

I hope this makes sense

**Comment by Pearlsky** [comment]
@TazDeCoder addressed requested changes, kindly review

**Review by TazDeCoder — APPROVED** [review]
Good job @Pearlsky!

**Pull request merged**